src/Palettes.md

```diff
@@ -22,72 +22,89 @@ Value | Color
+accessed directly, but instead through the following registers:
+
 ### FF68 - BCPS/BGPI (Background Color Palette Specification or Background Palette Index) - CGB Mode Only
 
-This register is used to address a byte in the CGBs Background Palette
-Memory. Each two byte in that memory define a color value. The first 8
-bytes define Color 0-3 of Palette 0 (BGP0), and so on for BGP1-7.
+This register is used to address a byte in the CGB's background palette RAM.
+Since there are 8 palettes, 8 palettes × 4 colors/palette × 2 bytes/color = 64 bytes
+can be addressed.
 
 ```
 Bit 7     Auto Increment  (0=Disabled, 1=Increment after Writing)
-Bit 5-0   Index (00-3F)
+Bit 5-0   Address ($00-3F)
 ```
 
-Data can be read/written to/from the specified index address through
-Register FF69. When the Auto Increment bit is set then the index is
-automatically incremented after each **write** to FF69. Auto Increment has
-no effect when **reading** from FF69, so the index must be manually
-incremented in that case. Writing to FF69 during rendering still causes
-auto-increment to occur.
+First comes BGP0 color #0, then BGP0 color #1, BGP0 color #2, BGP0 color #3,
+BGP1 color #0, and so on. Thus, address $03 allows accessing the second (upper)
+byte of BGP0 color #1 via BCPD, which contains the color's blue and upper green bits.
+
+Data can be read from / written to the specified CRAM address through
+BCPD/BGPD. If the Auto Increment bit is set, the index is
+automatically incremented after each **write** to BCPD. Auto Increment has
+no effect when **reading** from BCPD, so the index must be manually
+incremented in that case. Writing to BCPD during rendering still causes
+auto-increment to occur, despite the write being blocked.
 
-Unlike the following, this register can be accessed outside VBlank and
-HBlank.
+Unlike BCPD, this register can be accessed outside VBlank and HBlank.
 
 ### FF69 - BCPD/BGPD (Background Color Palette Data or Background Palette Data) - CGB Mode Only
 
-This register allows to read/write data to the CGBs Background Palette
-Memory, addressed through Register FF68. Each color is defined by two
-bytes (Bit 0-7 in first byte).
+This register allows to read/write data to the CGBs background palette memory,
+addressed through BCPS/BGPI. Each color is stored as little-endian RGB555:
 
 ```
-Bit 0-4   Red Intensity   (00-1F)
-Bit 5-9   Green Intensity (00-1F)
-Bit 10-14 Blue Intensity  (00-1F)
+Bit 0-4   Red Intensity   ($00-1F)
+Bit 5-9   Green Intensity ($00-1F)
+Bit 10-14 Blue Intensity  ($00-1F)
 ```
 
-Much like VRAM, data in Palette Memory cannot be read/written during the
-time when the LCD Controller is reading from it. (That is when the STAT
-register indicates Mode 3). Note: All background colors are initialized
-as white by the boot ROM, but it's a good idea to initialize at least
-one color yourself (for example if you include a soft-reset mechanic).
+Much like VRAM, data in palette memory cannot be read/written during the time
```

Comment:
```suggestion
Much like VRAM, data in palette memory cannot be read or written during the time
```

I'd limit the use of slashes. They are for informal writing (which our is not) and we already over use them